tutormfe/plugin.py

```diff
@@ -26,38 +26,47 @@
 CORE_MFE_APPS: dict[str, MFE_ATTRS_TYPE] = {
     "authn": {
         "repository": "https://github.com/openedx/frontend-app-authn",
+        "refs": "https://api.github.com/repos/openedx/frontend-app-authn/git/refs/tags",
```

Comment:
These default `refs` lines should probably be removed before merging; they're here just as a basis for discussion.  (Basically, it doesn't make much sense to use this feature with tags, as they're immutable anyway.)

Reply:
I think it does make sense to keep these references, because they are also used in the nightly branch, which makes use of a branch, and not a tag.

Reply:
Ok!  The only thing is that in the nightly branch we need to remember to change `tags` to `branches`:

```suggestion
        "refs": "https://api.github.com/repos/openedx/frontend-app-authn/git/refs/branches",
```

Reply:
Hmmm almost. It should be "https://api.github.com/repos/openedx/frontend-app-authn/git/refs/heads/master" in nightly. Thus the `MFE_COMMON_VERSION` should not be appended to that url in nightly (because "nightly != master"). Also, notice it's "heads" and not "branches".